ProjectLighthouse.Servers.GameServer/Controllers/Slots/PublishController.cs

```diff
@@ -46,14 +45,22 @@ public async Task<IActionResult> StartPublish()
 
         if (string.IsNullOrEmpty(slot.ResourceCollection)) slot.ResourceCollection = slot.RootLevel;
 
+        LbpFile? rootLevel = LbpFile.FromHash(slot.RootLevel);
+
+        if (rootLevel == null) return this.BadRequest();
+
+        GameVersion slotVersion = FileHelper.ParseLevelVersion(rootLevel);
+
+        if (slotVersion == GameVersion.Unknown) slotVersion = gameToken.GameVersion;
```

Comment:
nit: can prolly be spaced better
```suggestion
        LbpFile? rootLevel = LbpFile.FromHash(slot.RootLevel);
        if (rootLevel == null) return this.BadRequest();

        GameVersion slotVersion = FileHelper.ParseLevelVersion(rootLevel);
        if (slotVersion == GameVersion.Unknown) slotVersion = gameToken.GameVersion;
```